tools/ci_build/github/azure-pipelines/linux-multi-gpu-tensorrt-ci-pipeline.yml

```diff
@@ -4,6 +4,6 @@ jobs:
     AgentPool : 'Linux-Multi-GPU'
     JobName: 'Linux_CI_Multi_GPU_TensorRT_Dev'
     # The latest TensorRT container only supports ubuntu20.04 and python 3.8
-    RunDockerBuildArgs: '-o ubuntu20.04 -d tensorrt -x "--enable_multi_device_test"'
+    RunDockerBuildArgs: '-o ubuntu21.07 -d tensorrt -x "--enable_multi_device_test"'
```

Comment:
why 21.07 ?  should remain 20.04    (21.07 container is still based on Ubuntu 20.04)
are you mixing up the image version vs the ubuntu version?

Reply:
Are we going to totally remove Ubuntu? I've heard we may still keep it in case users want it. That's why I update to latest Nvidia NGC container, which supports TRT8.

Reply:
no, we shouldn't remove ubuntu. we can set up pipelines that still use it.
but we should fix the line 7 above,  -o ubuntu21.07 is wrong.
the option is for the run_dockerbuild.sh

Reply:
oh, yes. I overlooked this. Thanks.

Reply:
we can fix this in a separate PR